### PR TITLE
Manage deployment of GTFS-rt

### PR DESCRIPTION
**Pull request by pbougue**
Add management of workers and beat + env var for gtfsrt

### CODE REVIEW

File: fabfile/fabfile.py [hunk 1]
```diff
@@ -105,14 +105,16 @@ def disable_node(self, node):
...
 
 
-def deploy_container_safe(server, node_manager):
+def deploy_container_safe(role_dir, server, node_manager):
     """ Restart kirin on a specific server,
         in a safe way if load balancers are available
     """
     with settings(host_string=server):
         node_manager.disable_node(server)
+        change_dir(role_dir)
```
Comment (antoine-de) [inline fabfile/fabfile.py]: hum I think there are context manager in fab (or in fabtool) to manage this

**Comment by antoine-de** [comment]
@kinnou02 do you want to review it? Else I can merge it

File: fabfile/fabfile.py [hunk 2]
```diff
@@ -105,23 +105,37 @@ def disable_node(self, node):
...
         test_deployment()
         node_manager.enable_node(server)
 
 
+def deploy_kirin_beat_container_safe(server, node_manager):
+    """ Restart kirin on a specific server,
+        in a safe way if load balancers are available
+    """
+    with settings(host_string=server):
+        node_manager.disable_node(server)
```
Comment (kinnou02) [inline fabfile/fabfile.py]: no need to remove it from the LB...

File: fabfile/fabfile.py [hunk 3]
```diff
@@ -105,23 +105,35 @@ def disable_node(self, node):
...
         node_manager.disable_node(server)
-        restart()
+        restart('docker-compose_kirin.yml')
         test_deployment()
         node_manager.enable_node(server)
 
 
+def deploy_kirin_beat_container_safe(server, node_manager):
+    """ Restart kirin on a specific server,
+        in a safe way if load balancers are available
```
Comment (antoine-de) [inline fabfile/fabfile.py]: unsynchronized comments

**Pull request merged**